docs/src/tty.rst

```diff
@@ -58,8 +58,13 @@ API
     `readable`, specifies if you plan on calling :c:func:`uv_read_start` with
     this stream. stdin is readable, stdout is not.
 
+    On Unix this function will try to open ``/dev/tty`` and use it if the passed file
+    descriptor refers to a TTY. This allows us to put the tty in non-blocking mode
+    without affecting other processes that share it with us.
```

Comment:
Maybe: "This lets libuv put the tty in non-blocking mode without affecting other processes that share the tty."?  Pronouns and all that, eh?